src/helpers/genericArticleHelper.js

```diff
@@ -44,35 +44,53 @@ const filter = (title, tag, author, keyword) => {
   if (!tag) delete queryParams.query['$tag.category$'];
 
   // eslint-disable-next-line no-nested-ternary
-  const filterQuery = (keyword) ? { ...queryParams.keyword } : author ? { ...queryParams.author }
-    : { ...queryParams.query };
+  const filterQuery = keyword
+    ? { ...queryParams.keyword }
+    : author
+      ? { ...queryParams.author }
+      : { ...queryParams.query };
 
   return {
-    include: [{ model: Users, as: 'author' }, { model: Articles, as: 'article' }, { model: Categories, as: 'tag' }],
-    where: (keyword || title || tag || author) ? {
-      ...filterQuery
-    } : undefined
+    include: [
+      { model: Users, as: 'author' },
+      { model: Articles, as: 'article' },
+      { model: Categories, as: 'tag' }
+    ],
+    where:
+      keyword || title || tag || author
+        ? {
+          ...filterQuery
+        }
+        : undefined
```

Comment:
If line 60 is not true, where would be undefined anyways so please change it to something like
```suggestion
const hasFilterQuery = keyword || title || tag || author;
...
where: hasFilterQuery && { ...filterQuery  }
...
```

Reply:
Done.